### DISCUSSION TITLE
Make the title repeated over consecutive slides

### DISCUSSION THREAD

**Discussion by abduazizR** [quarto-dev/quarto-cli#11082] [Q&A]
Hi

Is it possible to repeat the title without having to rewrite it multiple times?

Example

```
---
title: "Presentation title"
author: Abdullah Abdelaziz, BPharm, MSc
format:
  revealjs:
    embed-resources: true
    footnotes-hover: true
    slide-number: c/t
bibliography: references.bib
---

In the current implementation I can do

## My first title

Content in slide 1

## My first title
Content in slide 2

## My first title
Content in slide 3


What I want to do is 

## My first title

Content in slide 1

Content in slide 2

Content in slide 3

```

I don't know how to make the title carry over multiple slides.

**Comment by mcanouil** [comment]
In the future use the Q&A category in Discussion when not reporting a bug.

**Comment by Steinthal** [comment]
Have a look at the [simplemenu extension](https://github.com/martinomagnifico/quarto-simplemenu?tab=readme-ov-file#simplemenu). 

It’s probably not the solution you’re looking for but it might give you some inspiration for a solution.

**Comment by mcanouil** [comment]
The only solution I see is to use a Lua filter that takes the previous title/header to fill the new one if empty.
